cypress/e2e/top100.cy.js

```diff
@@ -0,0 +1,182 @@
+      .and('have.css', 'fill', 'rgb(225, 46, 46)')
+    })
+
+    cy.get('#fav-button').click()
+    cy.get('.fav-game-card-wrapper').contains('Root')
+  })
+
+  it('Should display top100 when top100 button is clicked from favorites', () => {
+    cy.get('.heart-container').eq(0).click()
+    cy.get('#fav-button').click()
+    cy.get('.fav-game-card-wrapper').contains('Root')
+
+    cy.get('#top100-button').click()
+    cy.get('.gameCard-container').should('have.length', 2)
+    cy.get('.gameCard-container').eq(0).should('contain', "Root")
+      .and('contain', '$48.00')
+      .and('contain', '4.06 / 5')
+      .and('contain', '404 reviews')
+      .within(() => {
+        cy.get('img').should('have.attr', 'src', "https://s3-us-west-1.amazonaws.com/5cc.images/games/uploaded/1540147295104")
+        cy.get('button').eq(1).should('exist')
+      })
+
+    cy.get('.gameCard-container').eq(1).should('contain', "Scythe")
+      .and('contain', '$54.92')
+      .and('contain', '4.22 / 5')
+      .and('contain', '753 reviews')
+      .within(() => {
+        cy.get('img').should('have.attr', 'src', "https://cdn.shopify.com/s/files/1/0513/4077/1515/products/scythe-board-game.jpg?v=1611090922")
+        cy.get('button').eq(1).should('exist')
+      })
+    
+  })
+  
+  it('Displays details when game is clicked', () => {
+    cy.intercept('https://api.boardgameatlas.com/api/search?ids=TAAifFP590&client_id=NO0Fq8pQcF', {
+      method:"GET",
+      fixture: "../fixtures/details.json"
+    })
+    cy.get('.gameCard-container').eq(0).click()
+    cy.get('.details-and-comment-container').should('exist')
+  })
+
+  it('Navigate back to top100 page when clicking top100 button in details page', () => {
+    cy.intercept('https://api.boardgameatlas.com/api/search?ids=TAAifFP590&client_id=NO0Fq8pQcF', {
+      method:"GET",
+      fixture: "../fixtures/details.json"
+    })
+    cy.get('.gameCard-container').eq(0).click()
+    cy.get('.details-and-comment-container').should('exist')
+    cy.get('#top100-button').click()
+    cy.get('.gameCard-container').should('have.length', 2)
+    cy.get('.gameCard-container').eq(0).should('contain', "Root")
+      .and('contain', '$48.00')
+      .and('contain', '4.06 / 5')
+      .and('contain', '404 reviews')
+      .within(() => {
+        cy.get('img').should('have.attr', 'src', "https://s3-us-west-1.amazonaws.com/5cc.images/games/uploaded/1540147295104")
+        cy.get('button').eq(1).should('exist')
+      })
+
+    cy.get('.gameCard-container').eq(1).should('contain', "Scythe")
+      .and('contain', '$54.92')
+      .and('contain', '4.22 / 5')
+      .and('contain', '753 reviews')
+      .within(() => {
+        cy.get('img').should('have.attr', 'src', "https://cdn.shopify.com/s/files/1/0513/4077/1515/products/scythe-board-game.jpg?v=1611090922")
+        cy.get('button').eq(1).should('exist')
+      })
+  })
+
+  it('Should be able to search for a specific game', () => {
+    cy.get('#search-input').type('Roo')
```

Comment:
I recommend verifying the value of the search box contains "Roo" after use types it in the element